gui/WorldView.cxx

```diff
@@ -1016,9 +1010,18 @@ void WorldView::setMesh(vtkSmartPointer<vtkPolyData> mesh)
   d->volumeOptions->setEnabled(true);
 
   this->setVolumeVisible(d->UI.actionShowVolume->isChecked());
+  d->volumeOptions->showColorizeSurfaceMenu(true);
```

Comment:
Should this be conditional on the mesh actually having color data?

Reply:
No this just enables the menu which allows you to apply color to the mesh.  If it doesn't come with color you can still apply it from images.

Reply:
Ah.

> this just enables the menu which allows you to apply color to the mesh.

I knew that...

> If it doesn't come with color you can still apply it from images.

...but I didn't know that. Makes sense now; thanks!